AnnoyancesFilter/sections/popups.txt

```diff
@@ -781,6 +781,8 @@ stackshare.io#%#//scriptlet('remove-cookie', 'pageViews')
 tvoemisto.tv##.community_promo
 sobesednik.ru##.navbar.fixed-bottom
 news.naver.com###channelRecommendLayer
+reddit.com#$?##SHORTCUT_FOCUSABLE_DIV > div:has(> div > div[aria-modal="true"]) { display: none !important; }
```

Comment:
too wide. May block any modal window.

Reply:
@Alex-302, i've changed selector a bit. There are no locales so that should work. Layout is obfuscated so thats the only way i see here.